src/shared/components/ToolBar/ToolBar.tsx

```diff
@@ -1,25 +1,45 @@
-import React from 'react'
+import React, { useEffect, useState, useMemo } from 'react'
 import {useNavigate} from "react-router-dom";
 import {paths} from '../../data/constants';
-import { StyledButton, StyledToolBar, StyledContainer } from './styles';
+import {
+  StyledButton,
+  StyledToolBar, 
+  StyledContainer, 
+  ImgContainer, 
+  ButtonsContainer
+} from './styles';
 import logo from '../../data/logo.svg';
+import { colors } from '../../data/constants';
 
-
-const ToolBar: React.FC<{}> = (props) => {
+const ToolBar: React.FC<{pageType: string}> = (props) => {
   const navigate = useNavigate();
+  const getHomeColor = (pageType: string) => {
+    if (props.pageType === 'Home'){
+      return colors.pressedPageButton;
+    }
+    else{
+      return colors.ToolBar;
+    }
+  }
+  const homeColor: string = useMemo(() => getHomeColor(props.pageType), [props.pageType]);
```

Comment:
unless reuse of getHomeColor, you can use useMemo for homeColor, and then do the getHomeColor functionality (and remove getHomeColor)